Dangerfile

```diff
@@ -1,40 +1,13 @@
-# Sometimes its a README fix, or something like that - which isn't relevant for
-# including in a CHANGELOG for example
+# Sometimes it's a README fix, or something like that - which isn't relevant for
+# including in a project's CHANGELOG for example
+declared_trivial = pr_title.include? "#trivial"
 
-has_app_changes = !modified_files.grep(/lib/).empty?
-has_test_changes = !modified_files.grep(/spec/).empty?
+# Make it more obvious that a PR is a work in progress and shouldn't be merged yet
+warn("PR is classed as Work in Progress") if pr_title.include? "[WIP]"
 
-if ["KrauseFx", "orta"].include?(pr_author) == false
-  warn "Author @#{pr_author} is not a contributor"
-  
-  if modified_files.include?("*.gemspec")
-    warn "External contributors should not modify gemspec files"
-  end
-end
+# Warn when there is a big PR
+warn("Big PR") if lines_of_code > 500
 
-work_in_progress_warning
-
-if has_app_changes && !has_test_changes
-  warn "Tests were not updated"
-end
-
-if pr_body.length < 5
-  fail "Please provide a summary in the Pull Request description"
-end
-
-declared_trivial = pr_title.include?("#trivial") || !has_app_changes
-if !modified_files.include?("CHANGELOG.md") && !declared_trivial
-  fail "Please include a CHANGELOG entry. \nYou can find it at [CHANGELOG.md](https://github.com/danger/danger/blob/master/CHANGELOG.md)."
-end
-
-### Oddly enough, it's quite possible to do some testing of Danger, inside Danger
-### So, you can ignore these, if you're looking at the Dangerfile to get ideas.
-
-# If these are all empty something has gone wrong, better to raise it in a comment
-if modified_files.empty? && added_files.empty? && deleted_files.empty?
-  fail "This PR has no changes at all, this is likely a developer issue."
-end
-
-protect_files(path: "danger.gemspec",
-           message: ".gemspec modified",
-        fail_build: false)
+# Don't let testing shortcuts get into master by accident
+fail("fdescribe left in tests") if `grep -r fdescribe specs/`.length > 1
+fail("fit left in tests") if `grep -r "fit specs/ `.length > 1
```

Comment:
Possibly we could add this version to a `example/xcode/Dangerfile` so it would serve as quick-start, and we'd do the same with other providers/integrations? But this change should be undone indeed.